src/prx_server.c

```diff
@@ -186,7 +333,7 @@ static void prx_server_worker(
 {
     prx_server_socket_t* next;
     struct hashtable_itr* itr;
-    io_message_t* closerequest;
+    io_message_t* close;
```

Comment:
The old variable name has the advantage, that it doesn't collide with the function int close() from <unistd.h>.